pkg/cmd/cli/cmd/importimage.go

```diff
@@ -67,45 +68,147 @@ func RunImportImage(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, arg
 		return err
 	}
 
+	insecure := cmdutil.GetFlagBool(cmd, "insecure")
```

Comment:
I'm debating whether to remove import-image altogether, but yes, deserves a follow up.